rag/advanced_rag/knowlege_compile/structure.py

```
@@ -1582,7 +1650,7 @@ async def merge_one_group(old_id: str, state: dict):
             for start in range(0, len(rewrite_batch), _ES_DEDUP_EMBED_BATCH_SIZE):
                 batch = rewrite_batch[start : start + _ES_DEDUP_EMBED_BATCH_SIZE]
                 vectors = await _struct_embed(embd_mdl, [_struct_payload_description(payload) for _, payload in batch])
-                rewritten = [_struct_rebuild_es_doc(payload, base, vector, base.get("source_chunk_ids") or [], preserve_id=True) for (base, payload), vector in zip(batch, vectors)]
+                rewritten = [_struct_rebuild_doc_storage_doc(payload, base, vector, base.get("source_chunk_ids") or [], preserve_id=True) for (base, payload), vector in zip(batch, vectors)]
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)